lib/ddtrace/transport.rb

```diff
@@ -42,7 +46,7 @@ class HTTPTransport
     private_constant :API
 
     def initialize(hostname, port, options = {})
-      api_version = options.fetch(:api_version, V3)
+      api_version = options.fetch(:api_version, V4)
```

Comment:
why the default is `V4` now? V4 endpoint is for distributed sampling that is disabled by default. Endpoint `V3` is the right one to use unless an explicit configuration is used.

Reply:
I think I misread the code here. It had looked like in `writer.rb` that the `api_version` was always being set to `V4` anyways. That's only true of priority sampling though, not the default case.

I'll change this back.